astropy/utils/decorators.py

```diff
@@ -318,6 +320,11 @@ def deprecated_renamed_argument(old_name, new_name, since,
         Warning to be issued.
         Default is `~astropy.utils.exceptions.AstropyDeprecationWarning`.
 
+    alternative : str, optional
+        An alternative function or class name that the user may use in
+        place of the deprecated object if ``new_name`` is None. The deprecation
```

Comment:
I think I'd also use this alternative if `new_name` is given, i.e., make this something that overrides `new_name` in the message.

Reply:
I think I'm with @MSeifert04 here, it's either to use the new name or why bother renaming rather than removing if there is a preferred alternative elsewhere?